src/app/components/CreateLifeComponent.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Col, Row } from 'antd';
+import { Formik, Field, Form } from 'formik';
+import { useCreateLifeMutation, CreateLifeMutationVariables } from '../api/index';
+
+interface LifeValues {
+    firstName: string;
+    lastName: string;
+    title: string;
+    description: string;
+    birthday: Date;
+    hobbies: string[];
+}
+
+const CreateLifeComponent = () => {
+    const [mutation] = useCreateLifeMutation();
+    const handleCreateLife = async (lifeValues: LifeValues) => {
+        const createLifeMutationVariables: CreateLifeMutationVariables = {
+            lifeInput: lifeValues,
+        };
+        await mutation({ variables: createLifeMutationVariables });
+    };
+
+    return (
+        <div>
+            <Formik
+                initialValues={{
+                    firstName: '',
+                    lastName: '',
+                    title: '',
+                    description: '',
+                    birthday: new Date(),
+                    hobbies: [],
+                }}
```

Comment:
You can memoized the value in here instead with `useMemo` for the `intialValues`